src/LiveDevelopment/Documents/HTMLDocument.js

```diff
@@ -261,10 +261,12 @@ define(function HTMLDocumentModule(require, exports, module) {
 
             if (edits.length > 0) {
                 console.warn("Browser DOM does not match after change: " + JSON.stringify(change));
-
-                edits.forEach(function (delta) {
-                    console.log(delta);
-                });
+                if(this._debug){
+                  edits.forEach(function (delta) {
+                    // Uncomment to see all delta values (issue-619)
+                    //console.log(delta);
```

Comment:
You can uncomment this and remove the comment above, since it's surrounded by the `if(_debug)` check now.